### PR TITLE
ignore .mount cgroups associated with container on systemd

### PR DESCRIPTION
**Pull request by derekwaynecarr**
I am working to get the node e2e tests to pass on a system using docker with the systemd driver.  The first hurdle I ran into was duplicate entries are being returned in the pod summary stats for each container since multiple cgroup paths were being correlated back to the core container.  In this case, 

The first path is the scope traditionally associated with the container:

```
0307 12:55:16.210107    1856 summary.go:189] STATS buildSummaryPods - key /system.slice/docker-37515a45e527fd66a5ab125f689d0113e2ded24bba6b25453d3d78f66fb1f932.scope
I0307 12:55:16.210115    1856 summary.go:196] STATS buildSummaryPods - ref {nginx-2040093540-7uo6d default a514f7cf-e48d-11e5-bd2c-28d2444e470d}
```

The second path is associated with a systemd mount that is created for each mount into the container.

```
I0307 12:55:16.210413    1856 summary.go:189] STATS buildSummaryPods - key /system.slice/var-lib-docker-devicemapper-mnt-37515a45e527fd66a5ab125f689d0113e2ded24bba6b25453d3d78f66fb1f932.mount
I0307 12:55:16.210416    1856 summary.go:196] STATS buildSummaryPods - ref {nginx-2040093540-7uo6d default a514f7cf-e48d-11e5-bd2c-28d2444e470d}
I0307 12:55:16.210420    1856 summary.go:205] STATS buildSummaryPods - podStats &{{nginx-2040093540-7uo6d default a514f7cf-e48d-11e5-bd2c-28d2444e470d} 2016-03-07 17:55:02.669481834 +0000 UTC [{nginx 2016-03-07 17:55:04.121735851 +0000 UTC 0xc820e70030 0xc820826100 0xc8208ab7a0 0xc8208ab760 []}] 0xc8208261c0 []}
I0307 12:55:16.210429    1856 summary.go:209] STATS containerName nginx
```

Related to https://github.com/kubernetes/kubernetes/issues/22652

### CODE REVIEW

**Comment by derekwaynecarr** [comment]
@vishh @dchen1107 - if you have recommendations for a better place in the code I can filter, I am happy to change.  This doesn't fully fix the e2e_node_test on my environment, but it gets me past the first hurdle...

**Comment by derekwaynecarr** [comment]
After this patch, the remaining blocker to getting node e2e tests passing on systemd using devicemapper is due to https://github.com/google/cadvisor/issues/959

**Comment by derekwaynecarr** [comment]
@k8s-bot test this issue #22659

**Comment by dchen1107** [comment]
LGTM Thanks for fixing this. cc @timstclair

File: pkg/kubelet/server/stats/summary.go [hunk 1]
```diff
@@ -198,8 +209,12 @@ func (sb *summaryBuilder) buildSummaryPods() []stats.PodStats {
 			podToStats[ref] = podStats
 		}
 
+		glog.V(4).Infof("STATS buildSummaryPods - podStats %v", podStats)
+
 		// Update the PodStats entry with the stats from the container by adding it to stats.Containers
 		containerName := dockertools.GetContainerName(cinfo.Spec.Labels)
+		glog.V(4).Infof("STATS buildSummaryPods - containerName %v", containerName)
```
Comment (timstclair) [inline pkg/kubelet/server/stats/summary.go]: I'm a little worried about these log messages becoming to spammy, even on V(4). Could you at least reduce it to a single log entry with all the required info?

**Comment by timstclair** [comment]
Thanks for the fix!

**Comment by derekwaynecarr** [comment]
@timstclair - I removed the log messages since they were vestigial to me investigating the root cause.

**Comment by dchen1107** [comment]
LGTM again!

**Pull request merged**